.travis.yml

```diff
@@ -267,11 +258,11 @@ matrix:
         - rm -rf htmldocs/{.buildinfo,.doctrees}
       deploy:
         provider: pages
-        skip-cleanup: true
-        github-token: $SECRET_DEPLOY_GITHUB_PAGES
-        local-dir: core_build/htmldocs
+        cleanup: true
```

Comment:
Before we were skipping the cleanup, so this should be set to `false` I suppose.

Reply:
Thanks for that! I have updated the file.